tlatools/org.lamport.tlatools/pom.xml

```diff
@@ -61,13 +61,13 @@
         <plugin>
 	    <groupId>org.apache.maven.plugins</groupId>
 	    <artifactId>maven-antrun-plugin</artifactId>
-	    <version>1.8</version>
+	    <version>3.1.0</version>
 	    <executions>
 	    	<execution>
 	    		<id>compile</id>
 	    		<phase>compile</phase>
 	    		<configuration>
-	    			<tasks>
+	    			<target>
```

Comment:
Why is this change here?

Reply:
The newer version of the ant plugin uses target instead of tasks, it's a breaking change. Otherwise is identical.

Reply:
Please capture this kind of context in git commit messages in the future.